clearwater-infrastructure/etc/bash.bashrc.clearwater

```diff
@@ -1,22 +1,30 @@
+# Read out the node types on the server from the installed packages
 type=$(. /etc/clearwater/config
        {
+         # List all installed packages for CentOS or Debian. rpm returns an unordered list, dpkg-query is already sorted
          if which yum > /dev/null 2>&1 ; then
-           rpm -qa --qf '%{NAME}\n'
+           rpm -qa --qf '%{NAME}\n' | sort | uniq
          else
            dpkg-query -W -f='${Package}\n'
          fi
        } |
-       egrep '^(bono|clearwater-sip-stress|ellis|homer|homestead|sprout|ralf)$' |
+       egrep '^(bono|clearwater-sip-stress|ellis|homer|homestead|sprout|stats-engine|ralf)$' |
+       # Replacements with customer-friendly names and concatenate with hyphens
```

Comment:
There are several reasons to do these replacements:
- `sipp` and `mvse` are shorter than their counterparts
- If you have a name including hyphens, it may mess up any logic that depends on hyphen-separation.

I would make sure you mention all of these reasons.